docs/users-manual/quick-start-guide.rst

```diff
@@ -28,6 +28,25 @@ This guide presumes that
   (seen on the display), and standard error (seen on the display) may still
   be used, but their contents will be redirected from/to files.
 
+What is a Job?
+--------------
+
+Job is a very specific term in HTCondor. A job is the atomic unit of work.
+A job may use multiple cores on a machine, but one job may not (in general)
+run across more than one machine.  To effectively user HTCondor, you will
+need to divide your total work (often called a workflow) into a number
+of jobs, which can run asynchronously with respect to each other, but which
+may be connected by input and output files.  Each job is described by a
+Job ClassAd, which is usually created by the system from a submit description file.
+HTCondor is a High Throughput system, which means it has been designed to 
+effectively manage hundreds of thousands of jobs.  Attributes of jobs that
+must be defined include the executable or script to run, the amount of memory, CPU
+and other machine resources it needs, and descriptions of the file inputs it need.
+The set of files used by a job is called the "sandbox".  There is an input sandbox,
+the input files that exist before a job starts; the output sandbox, the set of files
+created by the job; and a scratch sandbox, the set of files made as the job runs.
```

Comment:
Don't we normally call the 'scratch sandbox' the 'job sandbox'?

Reply:
Feel like "job sandbox" is a bit confusing wrt the various places a sandbox can live.